plugins/repository-gcs/qa/google-cloud-storage/build.gradle

```diff
@@ -86,6 +86,19 @@ task createServiceAccountFile() {
 }
 
 task thirdPartyTest (type: Test) {
+    if (useFixture) {
+        thirdPartyTest.dependsOn createServiceAccountFile
```

Comment:
I'm wondering if  `dependsOn tasks.postProcessFixture` is also needed?

Reply:
Nope. The test fixtures plugin automatically makes all `Test` tasks depend on the fixture tasks. Essentially, when a project "uses a fixture" it's assumed that all tests require that fixture.

Reply:
Ok, thanks @mark-vieira !